plugin/doc_comment.js

```diff
@@ -333,7 +333,6 @@
           var key = parseType(scope, str, pos + 2);
           if (!key) return null;
           pos = skipSpace(str, key.end);
-          madeUp = madeUp || key.madeUp;
```

Comment:
It's a local variable, and it is reassigned [a few lines down](https://github.com/ternjs/tern/blob/master/plugin/doc_comment.js#L341) without any intervening reads. Or perhaps that assignment was mean to be of the form `madeUp = madeUp || ...` instead?